lms/templates/fields/field_image.underscore

```diff
@@ -0,0 +1,15 @@
+<div class="image-wrapper">
+    <img class="image-frame" src="<%- gettext(imageUrl) %>"  alt="<%=imageAltText%>"/>
+    <div class="u-field-actions">
+        <a class="u-field-upload-button">
+            <div class="upload-button-icon" aria-hidden="true"><%= uploadButtonIcon %></div>
+            <div class="upload-button-title" aria-live="polite"><%= uploadButtonTitle %></div>
+            <input class="upload-button-input" type="file" name="<%= id %>"/>
```

Comment:
@muhammad-ammar you can't have an ```<input>``` as a child of ```<a>``` so you'll want to move this down so that it is a sibling.